wykop/core/credentials.py

```diff
@@ -1,16 +1,27 @@
 class Credentials:
 
-    def __init__(self, appkey, secretkey, accountkey=None):
+    def __init__(self, appkey, secretkey, accountkey=None, login=None, password=None):
         self.appkey = appkey
         self.secretkey = secretkey
         self.account_key = accountkey
+        self.login = login
```

Comment:
Wydaje mi się, że watro też dodać te parametry do `__eq__`. Póki co jedyne miejsce które z tego korzysta (`MultiKeyWykopAPI`) raczej nie będzie wspirał tego logowania (chyba, że ruszy z marszu, sprawdzę później) ale warto zrobic to na zapas.